app/products/calls/screens/participants_list/participant.tsx

```diff
@@ -0,0 +1,159 @@
+
+import {useCurrentCall} from '@calls/state';
+import CompassIcon from '@components/compass_icon';
+import Emoji from '@components/emoji';
+import ProfilePicture from '@components/profile_picture';
+import Tag from '@components/tag';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+import {displayUsername} from '@utils/user';
+
+import type {CallSession} from '@calls/types/calls';
+
+const PROFILE_SIZE = 32;
+
+type Props = {
+    sess: CallSession;
+    teammateNameDisplay: string;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    rowContainer: {
+        flexDirection: 'row',
+        paddingTop: 8,
+        paddingBottom: 8,
+        gap: 16,
+        alignItems: 'center',
+    },
+    row: {
+        flexDirection: 'row',
+        flex: 1,
+        gap: 8,
+    },
+    picture: {
+        borderRadius: PROFILE_SIZE / 2,
+        height: PROFILE_SIZE,
+        width: PROFILE_SIZE,
+    },
+    name: {
+        ...typography('Body', 200, 'SemiBold'),
+        color: theme.centerChannelColor,
+        flex: 1,
+    },
+    you: {
+        ...typography('Body', 200, 'SemiBold'),
+        color: changeOpacity(theme.centerChannelColor, 0.56),
+    },
+    profileIcon: {
+        color: changeOpacity(theme.buttonColor, 0.56),
+    },
+    icons: {
+        flexDirection: 'row',
+        gap: 16,
+    },
+    muteIcon: {
+        color: changeOpacity(theme.centerChannelColor, 0.40),
+    },
+    unmutedIcon: {
+        color: changeOpacity(theme.centerChannelColor, 0.56),
+    },
+    hostTag: {
+        paddingVertical: 4,
+    },
+    raiseHandIcon: {
+        color: theme.awayIndicator,
+    },
+    screenSharingIcon: {
+        color: theme.dndIndicator,
+    },
+}));
+
+export const Participant = ({sess, teammateNameDisplay}: Props) => {
+    const intl = useIntl();
+    const currentCall = useCurrentCall();
+    const theme = useTheme();
+
+    const styles = getStyleSheet(theme);
+
+    if (!currentCall) {
+        return null;
+    }
+
+    return (
+        <Pressable
+            key={sess.sessionId}
+            testID='users-list'
+            style={styles.rowContainer}
+        >
+            {sess.userModel ? (
+                <ProfilePicture
+                    author={sess.userModel}
+                    size={PROFILE_SIZE}
+                    showStatus={false}
+                    url={currentCall.serverUrl}
+                />
+            ) : (
+                <CompassIcon
+                    name='account-outline'
+                    size={PROFILE_SIZE}
+                    style={styles.profileIcon}
+                />
+            )}
+            <View style={styles.row}>
+                <Text
+                    style={styles.name}
+                    numberOfLines={1}
+                >
+                    {displayUsername(sess.userModel, intl.locale, teammateNameDisplay)}
```

Comment:
`sess.userModel` won't update the details of the user model after fetched the first time, is that ok? do we expect to see the user display name changed real time if the user make changes to their profile? if the answer is yes, then we should observe the userModel for changes and if that is the case, same applies to the profile picture

Reply:
Thanks, good catch. I've made the change here: https://github.com/mattermost/mattermost-mobile/pull/7905/commits/194cd7d609583235c2180ea94cf2b6bfc583f6c2
Removing the memo only causes the sort to rerun one more time (when a participant changes their name or profile picture), which is fine -- we already observe those few fields from their user model in the `observeCurrentSessionsDict` observer.